libs/api/domains/user-profile/src/lib/userProfile.service.ts

```diff
@@ -62,14 +62,17 @@ export class UserProfileService {
 
       return {
         ...profile,
-        // Temporary solution while we still run the old user profile service.
+        /*
+          Temporary solution while we still run the old user profile service.
+          The Islyklar db will only be updated with a verified email. (See user-profile/verification.service.ts)
+          When in the state of an unverified email: Display the userprofile mail.
+        */
         mobilePhoneNumber:
           islyklarData?.mobilePhoneNumber ?? profile.mobilePhoneNumber,
-        email: islyklarData?.email ?? profile.email,
-        mobilePhoneNumberVerified: islyklarData?.mobilePhoneNumber
-          ? true
-          : profile.mobilePhoneNumberVerified,
-        emailVerified: islyklarData?.email ? true : profile.emailVerified,
+        email:
+          profile.emailVerified === false
+            ? profile.email
```

Comment:
What is the default value of `profile.emailVerified` for newly signed up users? If false, then this API will return an undefined email, right?

Is the plan now to have all island.is users verify their existing email when they sign in?

Reply:
The default value is false I believe.
Yes the plan is to have everyone who enters their email to verify it.

So the way this is set up:

User enters email in input
email is then saved with the userprofile service + a verification email is sent.
A get request for the user email would return the userprofile email at this point + `emailVerified: false`.

Once the user verifies, only then the email would be saved in the islyklar db. 
A get request at this point would return the verified email (saved in the islyklar db) + `emailVerified: true`